main/qml/Window.qml

```diff
@@ -159,20 +148,20 @@ ApplicationWindow {
         }
     }
 
-    property Tab currentTab: mainView.currentTab()
-    
+    property Tab currentTab: mainView.currentTab
+    property View currentView: mainView.currentView
 
     Item {
         anchors.fill: parent
         Keys.onPressed: {
-            var v = view(); if (v === undefined) return;
-            v.ctrl = (event.key == Qt.Key_Control) ? true : false;
+            var v = currentView; if (v === undefined) return;
+            if (event.key == Qt.Key_Control) v.ctrl = true;
             event.accepted = frontend.handleInput(event.text, event.key, event.modifiers)
             event.accepted = true;
```

Comment:
Oh I see. Like `event.preventDefault()` in JS. I guess my question was, why set `event.accepted` to the output of `frontend.handleInput`, but then override it with `true`. It sounds like `frontend.handleInput` should just return whether it's handled it or not.